tests/posix_time/tests/01-run.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim:fenc=utf-8
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+import time
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+US_PER_SEC = 1000000
+EXTERNAL_JITTER = 0.15
+
+class InvalidTimeout(Exception):
+    pass
+
+def testfunc(child):
+    try:
+        child.expect_exact("Please hit any key and then ENTER to continue")
+        child.sendline("a")
+        start_test = time.time()
+        child.expect_exact("5 x usleep(i++ * 500000)")
+        for i in range(5):
+            child.expect_exact("wake up")
+        child.expect_exact("5 x sleep(i++)")
+        for i in range(5):
+            child.expect_exact("wake up")
+        child.expect_exact("DONE")
+        testtime = (time.time() - start_test) * US_PER_SEC
+        exp = sum(i * 500000 for i in range(5)) + \
+              sum(i * US_PER_SEC for i in range(5))
+        lower_bound = exp - (exp * EXTERNAL_JITTER)
+        upper_bound = exp + (exp * EXTERNAL_JITTER)
+        if not (lower_bound < testtime < upper_bound):
```

Comment:
this behaviour is not documented anywhere, i.e., its not clear that this test will also test timer jitter and fail on some arbitrary bounds. This tests is missing a description (e.g. via README.md) of its purpose anyway, at least if its designed to do more, then simply checking that POSIX sleep and usleep do work.

Reply:
Will provide one (but this wasn't a requirement for `tests/xtimer_usleep`).

Reply:
README provided.

Reply:
actually I wasn't aware that such is done for `xtimer_usleep`, still short README is always nice to have.